receiver/datadogreceiver/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.22.0
 
 require (
 	github.com/DataDog/agent-payload/v5 v5.0.130
+	github.com/DataDog/datadog-agent/pkg/obfuscate v0.55.3
```

Comment:
Because this component is included by other ones (like tests and the dev distribution), you need to run `make gotidy`. Running it here, it yielded these changes:

```
> git st
On branch datadog-receiver/info-endpoint
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   receiver/datadogreceiver/go.sum
	modified:   testbed/go.mod
	modified:   testbed/go.sum
```

Reply:
Hmm, good to know, thanks :)